components/layout/footer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Typography } from '@interest-protocol/ui-kit';
+import { FC } from 'react';
+
+import { HeartSVG, IPXSVG } from '@/components/svg';
+
+import SocialSection from './socials-section';
+
+const Footer: FC = () => (
+  <Box
+    fontSize="s"
+    display="flex"
+    flexDirection="column"
+    textTransform="uppercase"
+  >
+    <Box textAlign="center" mb="l">
+      <Typography size="large" variant="label" fontWeight="500">
+        This DApp is maintained &
+      </Typography>
+      <Typography
```

Comment:
I believe use text-align center is a better approach than use display cuz this element is like text tag (h and p)